vendor.mod

```diff
@@ -28,6 +28,7 @@ require (
 	github.com/opencontainers/go-digest v1.0.0
 	github.com/opencontainers/image-spec v1.0.2
 	github.com/pkg/errors v0.9.1
+	github.com/rootless-containers/rootlesskit v0.14.6 // indirect
```

Comment:
This is something we need to fix in moby, as we shouldn't get this additional dependency.

I went looking "why" we get this dependency, and it looks like [`GetRootlessKitClient()`](https://github.com/moby/moby/blob/6b9b445af6c7908992632cff6c30cbf6a4c617ac/rootless/rootless.go#L31-L39) uses this dependency. While that function is only used on the daemon side, it's in the same package as [`RunningWithRootlessKit()`](https://github.com/moby/moby/blob/6b9b445af6c7908992632cff6c30cbf6a4c617ac/rootless/rootless.go#L22-L29), which is used in the CLI through [`registry.CertsDir()`](https://github.com/moby/moby/blob/2b70006e3bfa492b8641ff443493983d832955f4/registry/config_unix.go#L13-L24), which is used by [`registry.HostCertsDir()`](https://github.com/moby/moby/blob/7cba4ffa30466156b0915e9effe0a5d8c085e75e/registry/registry.go#L18-L21) which is called in various locations.

Reply:
do you want to postpone this PR waiting for this to be fixed?